tests/layouts/CoreLayout.spec.js

```diff
@@ -34,11 +34,4 @@ describe('(Layout) Core', function () {
     expect(_component.type).to.equal('div')
   })
 
-  it('Should render a child component.', function () {
```

Comment:
Not at all, was just making sure. And I'm not totally caught up to speed here: A regular React component class rendered inside of a pure component will still work w/ HMR, right?